### PR TITLE
Added documentation for autocatalogs

### PR DESCRIPTION
**Pull request by Moliholy**

### CODE REVIEW

**Comment by Moliholy** [comment]
I wonder if this PR should be merged in a different branch...

File: cpt-repo.tex [hunk 1]
```diff
@@ -455,6 +455,32 @@ \subsection{Managing Nested Catalogs with \texttt{.cvmfsdirtab}}
...
+  \item If the weight is less than \texttt{CVMFS\_AUTOCATALOGS\_MIN\_WEIGHT}, this catalog will be merged into its parent.
+\end{itemize}
+For instance, a possible configuration might be:
+\begin{verbatim}
+CVMFS_AUTOCATALOGS=true
+CVMFS_AUTOCATALOGS_MAX_WEIGHT=100000
+CVMFS_AUTOCATALOGS_MIN_WEIGHT=1000
+\end{verbatim}
+
+Note that both \texttt{CVMFS\_AUTOCATALOGS\_MAX\_WEIGHT} and \texttt{CVMFS\_AUTOCATALOGS\_MIN\_WEIGHT} have reasonable defaults and do not need to be defined by the user.
```
Comment (DrDaveD) [inline cpt-repo.tex]: I'd like to see the defaults explicitly listed in the documentation.  

Also, I'll note that this section will also need to go into http://cernvm.cern.ch/portal/filesystem/maintain-repositories after it is released.

**Comment by Moliholy** [comment]
We would like to have the "flexibility" to change these defaults at any point, specially once we have some user feedback. So far anyway the parameters are 100000 and 1000, which attending at the most important repositories (atlas, cms, alice...) look reasonable.

**Comment by DrDaveD** [comment]
Sure, but it would have to be in combination with a new cvmfs release, and there's always a new techreport with a new cvmfs release, so the documention can be changed accordingly.  I think it's important to tell users what defaults are.

**Comment by jblomer** [comment]
I think it makes sense to document the default values.  We have the documentation versioned and aligned with the releases, so if we decide to change it, we can as well change the documentation.

**Comment by Moliholy** [comment]
Defaults added

**Pull request merged**